source/shared/net/references/manage-accounts.rst

```diff
@@ -115,14 +112,21 @@ also be set manually via the option ``--nonce``.
 Tools
 =====
 
+The Desktop Wallet
+------------------
+
+The Desktop Wallet is a digital wallet that enables you to create and manage your Concordium identities, credentials, and accounts from you desktop and to create transactions such as sending CCD, adding a baker, and exporting and importing account information.
+
 The Mobile Wallet
```

Comment:
Should we maybe rewrite this section? It sounds a bit like identities cannot be made in the mobile wallet. 

Maybe something a la:
_The Mobile Wallet is a digital smartphone wallet, that enables you to create and manage your Concordium identities and accounts, to create simple and shielded transactions, and to export and import your accounts and identities._

What do you think?